### DISCUSSION TITLE
Fish detection

### DISCUSSION THREAD

**Discussion by atv2016** [blakeblackshear/frigate#6773] [General]
I don't how frigate+ is coming along, but i would love to be able to detect fish in the pond with a underwater camera i have. That would be super useful for analytics.

I don't see it in the objects it can detect so it will have to be a custom model? Still a fish should be a fairly easily recognisable model (one would think!).

Love to hear thoughts about this.

**Comment by mitchins** [comment]
I'd also love this.
Given the new Frigate+ it's possible to probably upload fish and train it.
On the other hand if not using coral but OpenVINO or CUDA, there may be models that support it however I am unsure if Frigate needs to support the label - probably it does.

**Comment by atv2016** [comment]
I just set person detection very low and that catches all the fish now.
________________________________

**Comment by dopeytree** [comment]
How is this working out today? I'm exploring options for a pond camera, I guess motion detection alone could work.. 
Also which camera did you go for?
